sdk/template-dpg/Azure.ServiceTemplate.Template/samples/TemplateClientSamples.md

```diff
@@ -0,0 +1,45 @@
+# Sample readme template
```

Comment:
Curious why we have this here?

It is unclear in this file which are instruction for the SDK author (the service team partner) and what is part of the Anomaly Detector README.  

This also doesn't follow our samples guidance.  Please see the example in https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/anomalydetector/Azure.AI.AnomalyDetector/samples 

The `<service>/<library>/samples` directory should contain:
- README.md
- Sample1_<scenario_name>.md
- ...
- Sample<number_of_samples>_<scenario_name>.md

Reply:
updated to consistent with the sdk-layout.